arglonbot/Program.cs

```diff
@@ -72,6 +72,19 @@ void Console_CancelKeyPress(object? sender, ConsoleCancelEventArgs e)
     cts.Cancel();
 }
 
+string Message() => Now() switch
+{
+    { Month: 12, Day: 24 } now => "Happy christmas eve 🎅🏻",
+    { Month: 12, Day: 25 } => "Happy christmas 🎅🏻",
```

Comment:
I'm both shocked and disturbed that the brits say "Happy christmas" and not "Merry Christmas"

Although the "night before christmas" traditional children's book I grew up with had "happy christmas" so I'll allow it